grails-app/services/au/org/ala/collectory/SitemapService.groovy

```diff
@@ -106,22 +112,23 @@ class SitemapService {
         countUrls++
     }
 
+    @Transactional
```

Comment:
Scheduled runs crash with *org.springframework.dao.DataAccessResourceFailureException: Could not obtain current Hibernate Session; nested exception is org.hibernate.HibernateException: No Session found for current thread*. Manual runs from admin UI are ok. Adding `@Transactional` fixes the crash. But there might be other/better solutions?